docs/cpu.md

```diff
@@ -0,0 +1,60 @@
+# CPU example usage
+
+This document describes an example use of the [cpu](https://github.com/u-root/cpu)
+command.
+
+## Introduction
+
+The `cpu` command allows you to log in to a remote system (in this case RTE) and
```

Comment:
@PLangowski The quality of this description could be a lot better. Can we not improve it? I think the main advantage of cpu is not that we can `see some files` from host PC. Please provide better description.

Reply:
done e2e288c2cfe632e4be4be87116ed66b448439d17
Is this ok?

Reply:
A lot better. Thanks